contracts/DAO.sol

```diff
@@ -403,7 +403,7 @@ contract DAO {
     function sendTreasuryFunds(uint256 amount, address payable destination) external tierTwoDelegation {
         Treasury ty = Treasury(trsy);
         uint256 toAdd = ty.sendFunds(amount, destination);
```

Comment:
Do you need to set it to the toAdd variable?

Reply:
no good catch